mls-rs/src/group/epoch.rs

```diff
@@ -30,6 +30,8 @@ pub(crate) struct PriorEpoch {
     pub(crate) self_index: LeafIndex,
     pub(crate) secrets: EpochSecrets,
     pub(crate) signature_public_keys: Vec<Option<SignaturePublicKey>>,
+    #[cfg(feature = "validate_public_message_prior_epoch")]
```

Comment:
I'm concerned about the format of stored data being different when the feature is on vs off. I can discuss with @mulmarta, but one suggestion is that we can add a feature to the mls encoding macro to have fields at the end of a struct be optional and automatically filled in with the Default value for that type. 

The end result is something like  this:

```
#[mls(default)]
membership_key: Option<Vec<u8>>
```

Reply:
Yeah, I had the same thoughts. MLS encoding is not great for these stored values. It would be better to have something more flexible.

Reply:
This makes sense. Is it something that we could do in a followup PR, since it sounds like some changes would be required to the MlsEncode trait?

Reply:
Good point, we do have to clean that up overall, can target for v 1.0